packages/lib-classifier/src/plugins/tasks/TextTask/models/TextAnnotation.js

```diff
@@ -0,0 +1,8 @@
+import { types } from 'mobx-state-tree'
+
+const TextAnnotation = types.model('TextAnnotation', {
+  task: types.identifier,
+  value: types.maybeNull(types.string)
```

Comment:
This should default to be a an empty string since this is what is used by the classifier to add a default annotation when a value isn't filled in and the task is not required. An empty string was used for the default (https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/classifier/tasks/text/index.jsx#L148) and I don't think there's any reason to change this.

So, `types.optional(types.string, '')` instead.

Reply:
Good point.